src/couch_db.erl

```diff
@@ -1560,6 +1561,30 @@ is_systemdb(DbName) when is_list(DbName) ->
 is_systemdb(DbName) when is_binary(DbName) ->
     lists:member(dbname_suffix(DbName), ?SYSTEM_DATABASES).
 
+format_db_record(#db{} = Db) ->
```

Comment:
should we make this more flexible by adding format_db_record/2 that takes in a separate list rather and then having it to default to this list when users don't provide the db keys they want? Just thinking longer term.